deploy/examples/nfs.yaml

```diff
@@ -47,11 +47,11 @@ spec:
     # Resource requests and limits to apply to NFS server pods
     resources:
       # limits:
-      #   cpu: "500m"
-      #   memory: "1024Mi"
+      #   cpu: "3"
+      #   memory: "8Gi"
       # requests:
-      #   cpu: "500m"
-      #   memory: "1024Mi"
+      #   cpu: "3"
+      #   memory: "8Gi"
```

Comment:
The latest recommendations we have for these numbers are 3 CPU cores and 8Gi of RAM, so I update that here with the design doc. Seems decently related.